guides/common/modules/proc_disabling-cockpit-on-project.adoc

```diff
@@ -4,20 +4,11 @@
 Perform the following procedure if you want to disable {Cockpit} on {Project}.
 
 .Procedure
-. Run this `{foreman-installer}` command:
+* Run this `{foreman-installer}` command:
```

Comment:
Actually, looking at the procedure for enabling cockpit, it has a more informative procedure step description:

`Enable RHEL web console on your Satellite Server:`

vs

`Run this foreman-installer command`

Can you think of a similar way to rephrase this step as well? Looking at the actual command here, I can't tell if simple "Disable RHEL web console on your Satellite Server:" would be accurate.

Reply:
There's quite a bit of overlap with line 4. I'll just rephrase this line now, but leave it up to other writers to decide what's best here in the long term.

Reply:
We discussed this at today's meeting and agreed that `Disable {Cockpit} on your {ProjectServer}:`, as included in the latest version of this PR, is okay.